### ISSUE TITLE
ovh.net

### ISSUE THREAD

**Issue by zadigre** [lightswitch05/hosts#142]
hello!

blocking ovh.net will block their Speedtest web page on http://ovh.net. 
I whitelisted this single host and all is fine. 

for me this is a false positive in your Ads & Tracking list.

**Comment by lightswitch05** [comment]
thank you for bringing this to my attention. I also believe this is a false positive.

**Comment by zadigre** [comment]
I'm also not 100% sure that all those ovh.net subdomains are really required. 
I didn't encounter any other problem so far... but I'm wondering why all those are present in your list..

**Comment by lightswitch05** [comment]
* `ovh.net` was falsely added by https://github.com/lightswitch05/hosts/commit/2b532611 - where I added a bunch of domains from a vendor tracking list. It should have never been added.
* I use a couple different tools to automatically find and add any subdomain of a blocked root domain. Since I had added `ovh.net` - the subdomains were added automatically. Removing the root domain will also remove the subdomains.
